.fides/fidesctl.toml

```diff
@@ -1,14 +1,24 @@
-[api]
-database_user = "postgres"
-database_password = "fidesctl"
-database_host = "localhost"
```

Comment:
weirdly I don't think this value was being used before....needs some looking into but the default fidesctl-db was being used here